javascript/javascript3/week2/badMovies.js

```diff
@@ -0,0 +1,25 @@
+const movie_Api = 'https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json'
+const badTitlesAfter2000 = []
+let ali = []
+
+async function badMovies(){
```

Comment:
function name shoudl start with a verb to describe the behaivure of the function.
for example:
findBadMovies(), 
getBadMovies(), 
UpdateBadMovies(), 
...